library/core/src/any.rs

```diff
@@ -610,6 +610,84 @@ impl dyn Any + Send + Sync {
 /// While `TypeId` implements `Hash`, `PartialOrd`, and `Ord`, it is worth
 /// noting that the hashes and ordering will vary between Rust releases. Beware
 /// of relying on them inside of your code!
+///
+/// **Note on Variance**:
+/// Suppose `SubType` is a subtype of `SuperType`,
+/// that is, `SubType` can be used wherever `SuperType` can be used,
+/// and can be freely as-casted to it,
```

Comment:
```suggestion
/// Suppose `SubType` is a subtype of `SuperType`, that is,
/// values of type `SubType` can be used wherever `SuperType` is
/// expected (so they can, for instance, be freely `as`-casted to it).
```

Reply:
My text is wrong, so thanks for catching this.